src/views/MarketPlace/components/MarketPlacePopup/index.jsx

```diff
@@ -172,9 +172,12 @@ Thank you
         .catch(() => setProfileImg(null));
 
       // Fetch profile info
-      getProfileInfo(item.PosterUsername)
-        .then((info) => {
-          setProfileInfo(info);
+      userService
+        .getInformalName(item.PosterUsername)
+        .then((profile) => {
+          setProfileInfo(
+            profile ? { NickName: profile.NickName, LastName: profile.LastName } : null,
+          );
```

Comment:
Since `getInformalName` returns a `{NickName, LastName}` object or undefined, isn't this functionally identical to:
```suggestion
        .then(setProfileInfo);
```